### ISSUE TITLE
When trying to upload to firefox shipit raises a TypeError

### ISSUE THREAD

**Issue by spookyuser** [LinusU/wext-shipit#8]
```
× TypeError: StreamSearch is not a constructor
TypeError: StreamSearch is not a constructor
```

It actually still successfully deploys. Btw I love the idea of this module so much!

**Comment by spookyuser** [comment]
I know why this is happening. `text-stream-search` is ^2.1.1 but it needs to be 2.1.1 exactly. I think that module underwent breaking api changes but did not bump the version number correctly. So if you install 2.1.1 exactly this fixes this.

**Comment by LinusU** [comment]
Ugh, this is annoying...

I'm trying to work on solving this upstream: https://github.com/Originate/node-text-stream-search/issues/31, https://github.com/Originate/node-text-stream-search/pull/32

I'll give them a little time to merge and release that, otherwise we'll pin the dependency here...

Sorry for any inconvenience!

**Comment by spookyuser** [comment]
IKR :P but no worries, i'm currently installing from a fork of this project so take your time.

**Comment by LinusU** [comment]
fixed so that it can import both: 933a088

**Comment by LinusU** [comment]
published `@wext/shipit@0.1.5`

**Comment by timbru31** [comment]
Unfortunately it fails again for me:

```
•100% ➜ npm run shipit:firefox

> amazon-tag-remover@0.5.2 shipit:firefox /Users/timbru/work/private/amazon-tag-remover
> shipit firefox dist

✖ TypeError: StreamSearch is not a constructor
TypeError: StreamSearch is not a constructor
    at module.exports (/Users/timbru/work/private/amazon-tag-remover/node_modules/@wext/shipit/lib/firefox.js:33:22)
    at main (/Users/timbru/work/private/amazon-tag-remover/node_modules/@wext/shipit/bin.js:27:11)
    at Object.<anonymous> (/Users/timbru/work/private/amazon-tag-remover/node_modules/@wext/shipit/bin.js:35:1)
    at Module._compile (internal/modules/cjs/loader.js:776:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:787:10)
    at Module.load (internal/modules/cjs/loader.js:643:32)
    at Function.Module._load (internal/modules/cjs/loader.js:556:12)
    at Function.Module.runMain (internal/modules/cjs/loader.js:839:10)
    at internal/main/run_main_module.js:17:11
Server response: Duplicate add-on ID found. (status: 400)
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! amazon-tag-remover@0.5.2 shipit:firefox: `shipit firefox dist`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the amazon-tag-remover@0.5.2 shipit:firefox script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/timbru/.npm/_logs/2019-07-18T11_46_37_837Z-debug.log
```

**Comment by filoxo** [comment]
I'm seeing this again, presumably because [TextStreamSearch no longer has a default export](https://github.com/kevgo/node-text-stream-search/blob/master/src/text-stream-search.ts#L10) and this package is [checking for a default export](https://github.com/LinusU/wext-shipit/blob/master/lib/firefox.js#L10) instead of a named export. One could patch this locally by adding this below *firefox.js:L10*:

```
if (StreamSearch.TextStreamSearch) StreamSearch = StreamSearch.TextStreamSearch;
```

Thoughts on reopening this issue until its fixed? I can open a PR in the meanwhile.

EDIT: Looks like the author of TextStreamSearch already submitted #14 to fix this 🎉

**Comment by LinusU** [comment]
Should be fixed in `0.2.1`, please let me know if it's still not working 🚀
